docs/data-sources/omim.mdx

```diff
@@ -0,0 +1,219 @@
+---
+title: OMIM
+---
+
+import JSON from './omim-json.md'
+
+## Overview
+
```

Comment:
In the overview , I usually like to provide a short introduction to the data source. Details of parsing and using the data source should be in a subsequent section in my opinion.

Reply:
I added a description of OMIM